README.md

```diff
@@ -10,6 +10,20 @@ Available tags:
 - dev-nginx
 - v0.12.4-nginx
 
+## Environment Variables
+
+In addition to [the environment variables supported by PixelFed](https://docs.pixelfed.org/running-pixelfed/configuration.html), the following variables are supported:
+
+| Variable | Description | Default |
+| -------- | ----------- | ------- |
+| APP_PORT | Port that nginx will listen on | - |
+| APP_DOMAIN | Server name/domain for nginx configuration | - |
+| COMPOSER_VERSION | Version of Composer to install | 2.5.5 |
+| COMPOSER_HOME | Path to Composer home directory | /var/www/.composer |
+| COMPOSER_MEMORY_LIMIT | Memory limit for Composer | -1 |
+| DEBIAN_FRONTEND | Debian frontend configuration | noninteractive |
```

Comment:
These are image build args. They're only used to setup the environment during the image build from the Dockerfile. The other env vars are used at runtime by the start script.

It's fine to include the build args as a separate table for anyone that wants to build their own image from this repo.

Reply:
Actually on second look, I'm wrong. They're `ENV`s so they get set as default variables during runtime. Only `COMPOSER_VERSION` looks like it would apply during the build and if we want that to be toggled, it would need swapped to an `ARG`.

Reply:
hmm you make a good point though that we probably wouldnt expect people to override them..

Should I just do `APP_PORT`, `APP_DOMAIN`, and `ENABLE_HORIZON`?

Reply:
Sure 👍🏻

Reply:
should be set!